manifests/component/action.pp

```diff
@@ -3,17 +3,19 @@
   String $target,
   String $confdir,
   String $type,
+  Boolean $dont_linebreak_actions = $rsyslog::dont_linebreak_actions,
```

Comment:
Considering a catalog with:

```puppet
rsyslog::component::action { 'foo':
  # ...
}
```

this will fail because the `rsyslog` was not included (it is included on line 10, but the parameter default value need it before that code is run).

The following would work:

```puppet
include rsyslog

rsyslog::component::action { 'foo':
  # ...
}
```

The test suite probably need to be adjusted to ensure the main class of the module is included before testing the `rsyslog::component::action` class:

https://rspec-puppet.com/documentation/defined_types/#specifying-code-to-include-before